pay-api/src/pay_api/services/statement.py

```diff
@@ -248,6 +248,7 @@ def get_invoices_owing_amount(auth_account_id: str):
                 .filter(InvoiceModel.invoice_status_code.in_((InvoiceStatus.SETTLEMENT_SCHEDULED.value,
                                                               InvoiceStatus.PARTIAL.value,
                                                               InvoiceStatus.CREATED.value,
```

Comment:
I think we need to take out CREATED

Reply:
The parent function doesn't have CREATED:

![image](https://github.com/user-attachments/assets/4ea36777-086a-47ec-8f2d-0b036bb4b8ae)